src/Metal/Defs.cs

```diff
@@ -226,6 +226,19 @@ public struct MTLDispatchThreadgroupsIndirectArguments {
 		public uint ThreadGroupsPerGrid3;
 	}
 
+	[NoMac]
+	[iOS (10,0), TV (10,0)]
+	public struct MTLStageInRegionIndirectArguments
+	{
+		public uint StageInOrigin1;
+		public uint StageInOrigin2;
+		public uint StageInOrigin3;
```

Comment:
it's
```
uint32_t stageInOrigin[3];
uint32_t stageInSize[3];
```
Maybe it would be better to use property over a private storage.
Let's start by double check the native vs managed size and alignment...

Reply:
I followed the wrong sheep then: https://github.com/xamarin/xamarin-macios/blob/xcode10/src/Metal/Defs.cs#L223-L227

```
typedef struct {
    uint32_t threadgroupsPerGrid[3];
} MTLDispatchThreadgroupsIndirectArguments;
```

Reply:
So same native and managed size (24)

Reply:
Oh and I forgot but the native alignment is 4 bytes so I believe we can go `[StructLayout (LayoutKind.Sequential)]`.